packages/components/src/MDXDeck.js

```diff
@@ -90,10 +93,12 @@ export class MDXDeck extends React.Component {
       }
     } else {
       switch (keyCode) {
+        case keys.pgDown:
```

Comment:
Thanks for this! I might be totally wrong, but shouldn't `pageDown` advance to the next slide?

Reply:
Good question. I'll ask the author of the issue.

Reply:
Based on what Google Slides does it seems to be page down for next slide